calrissian/job.py

```diff
@@ -122,7 +122,11 @@ def add_persistent_volume_entries_from_pod(self, pod):
             # Mount any additional volumes not already mounted, only duplicate is "calrissian-wdir"
             if claim_name != "calrissian-wdir":
                 log.info(f"Adding PVC {claim_name} mounted at {mount_path}")
-                self.add_volume_binding(mount_path, mount_path, False)
+                writable = False
+                # Need to allow writing to the workspace directory
+                if claim_name.startswith("pvc-"):
+                    writable = True
+                self.add_volume_binding(mount_path, mount_path, writable=writable)
```

Comment:
I'm wondering if something like this might offer any advantages:


```suggestion
                # Need to allow writing to the workspace directory
                writable = claim_name.startswith("pvc-")
                self.add_volume_binding(mount_path, mount_path, writable=writable)
```

Reply:
Good idea, added